src/Tools/ExternalAccess/Razor/Features/Cohost/Constants.cs

```diff
@@ -11,6 +11,8 @@ internal static class Constants
 {
     public const string RazorLanguageName = LanguageInfoProvider.RazorLanguageName;
 
-    // The UI context is provided by Razor, so this guid must match the one in https://github.com/dotnet/razor/blob/main/src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/RazorConstants.cs
+    // These UI contexts are provided by Razor, so must match https://github.com/dotnet/razor/blob/main/src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/RazorConstants.cs
```

Comment:
Should Razor simply use these values from the EA?

Reply:
Yeah maybe not worth the trouble then. Is there a unit test in razor to ensure these are in sync?